src/routes/AgreementLineViewRoute/AgreementLineViewRoute.test.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '@folio/stripes-erm-components/test/jest/__mock__';
+import { renderWithIntl } from '@folio/stripes-erm-components/test/jest/helpers';
+import { MemoryRouter } from 'react-router-dom';
+import { noop } from 'lodash';
+import { Button } from '@folio/stripes/components';
+import { Button as ButtonInteractor } from '@folio/stripes-testing';
+import {
+  match,
+  line,
+  orderLines,
+  query
+} from './testResources';
+import translationsProperties from '../../../test/helpers';
+import AgreementLineViewRoute from './AgreementLineViewRoute';
+
+const CloseButton = (props) => {
+  return <Button onClick={props.handlers.onClose}>CloseButton</Button>;
+};
+
+const EditButton = (props) => {
+  return <Button onClick={props.handlers.onEdit}>EditButton</Button>;
+};
+
+const ToggleTagsButton = (props) => {
+  return <Button onClick={props.handlers.onToggleTags}>ToggleTagsButton</Button>;
+};
+
+CloseButton.propTypes = {
+  handlers: PropTypes.shape({
+    onClose: PropTypes.func,
+  }),
+};
+
+EditButton.propTypes = {
+  handlers: PropTypes.shape({
+    onEdit: PropTypes.func,
+  }),
+};
+
+ToggleTagsButton.propTypes = {
+  handlers: PropTypes.shape({
+    onToggleTags: PropTypes.func,
+  }),
+};
+
+const historyPushMock = jest.fn();
+const mutaturAgrPutMock = jest.fn();
```

Comment:
Can we rename this mutatorAgreementPutMock (Or whatever the lengthened version of Agr is?) to avoid future confusion?
Also mutator instead of mutatur